packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1172,6 +1172,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // The last message processed at the time of summary. If there are no messages, nothing has changed from
             // the base summary we loaded from. So, use the message from its metadata blob.
             message: extractSummaryMetadataMessage(this.deltaManager.lastMessage) ?? this.baseSummaryMessage,
+            runtimeVersion: pkgVersion,
```

Comment:
Looking at how it's used (ContainerRuntime.shouldWriteMetadata), this data is written into every snapshot, so I'd call properties to better reflect their meaning (i.e. it's runtime of a client that produced that summary, as opposed to let's say runtime that created that file in the first place).
So it's useful and I'll add it (under different name), but I'd also want to see a set of properties (like runtime version, time stamp) that are created only at file creation time, and are never changed after that (i.e. simply preserved from snapshot to snapshot without modification).